source/runner/DangerUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function sentence(array: Array<string>): string {
```

Comment:
I don't mind the if JSDoc for them is extremely limited ( if for example we had to add it due to tslint ) mainly because the type system would keep us in line as you say